CMakeLists.txt

```diff
@@ -204,7 +229,11 @@ IF(BOOST_STATIC)
     set(Boost_USE_STATIC_LIBS   ON)
 endif()
 
-find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
+if (USE_QT)
+    find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgQt osgUtil osgFX)
+else()
+    find_package(OpenSceneGraph 3.2.0 REQUIRED osgDB osgViewer osgText osgGA osgAnimation osgParticle osgUtil osgFX)
+endif()
```

Comment:
Use an intermediate variable here to avoid the duplication of library lists. Suggestion in comment https://github.com/OpenMW/openmw/pull/835#discussion_r46482768